diff --git a/node_modules/@types/proj4/index.d.ts b/node_modules/@types/proj4/index.d.ts
index d7d4cf1..d1edc36 100644
--- a/node_modules/@types/proj4/index.d.ts
+++ b/node_modules/@types/proj4/index.d.ts
@@ -112,5 +112,6 @@ declare function proj4<T extends proj4.TemplateCoordinates>(
     coordinates: T
 ): T;
 
-export = proj4;
+// export = proj4;
+export default proj4;
 export as namespace proj4;
